servers/annotations-api/package.json

```diff
@@ -47,7 +47,8 @@
     "nock": "13.4.0",
     "nodemon": "3.0.2",
     "supertest": "6.3.3",
-    "ts-jest": "29.1.1"
+    "ts-jest": "29.1.1",
+    "jest-extended": "4.0.2"
```

Comment:
@kschelonka does this need to be added to setup?

Reply:
maybe, although everything works for me locally (but it could be pulling my setup from somewhere).

Reply:
oh, it was already set up. must have been copied before

Reply:
Oh. I wonder if we are hoistng jest in .npmrc